test/validate.spec.js

```diff
@@ -411,6 +418,36 @@ describe('screener-runner/src/validate', function() {
       });
     });
 
+    describe('validate.sauce', function() {
+      it('should forbid tunnelIdentifier when launchSauceConnect is true', function() {
+        return Validate.runnerConfig({apiKey: 'key', projectRepo: 'repo', states: [], sauce: {username: 'user', accessKey: 'key', launchSauceConnect: true, tunnelIdentifier: 'id'}})
+          .catch(function(err) {
+            expect(err.message).to.equal('child "sauce" fails because [child "tunnelIdentifier" fails because ["tunnelIdentifier" is not allowed]]');
+          });
+      });
+
+      it('should forbid tunnelIdentifier when launchSauceConnect is true', function() {
+        return Validate.runnerConfig({apiKey: 'key', projectRepo: 'repo', states: [], tunnel: {host: 'host'}, sauce: {username: 'user', accessKey: 'key', launchSauceConnect: true}})
+          .catch(function(err) {
+            expect(err.message).to.equal('"tunnel" is not allowed');
+          });
+      });
+
+      it('should forbid screener browsers when launchSauceConnect is true', function() {
+        return Validate.runnerConfig({apiKey: 'key', projectRepo: 'repo', states: [], browser: [{browserName: 'chrome'}], sauce: {username: 'user', accessKey: 'key', launchSauceConnect: true}})
+          .catch(function(err) {
+            expect(err.message).to.equal('"browser" is not allowed');
```

Comment:
@xwa130saucer is it possible to update some of these error messages to be more friendly? Does Joi allow us to do that?